core/src/test/java/org/apache/iceberg/TestTableMetadata.java

```diff
@@ -160,31 +155,16 @@ public void testJsonConversion() throws Exception {
                 .build());
 
     TableMetadata expected =
-        new TableMetadata(
-            null,
-            2,
-            UUID.randomUUID().toString(),
-            TEST_LOCATION,
-            SEQ_NO,
-            System.currentTimeMillis(),
-            3,
-            7,
-            ImmutableList.of(TEST_SCHEMA, schema),
-            5,
-            ImmutableList.of(SPEC_5),
-            SPEC_5.lastAssignedFieldId(),
-            3,
-            ImmutableList.of(SORT_ORDER_3),
-            ImmutableMap.of("property", "value"),
-            currentSnapshotId,
-            Arrays.asList(previousSnapshot, currentSnapshot),
-            null,
-            snapshotLog,
-            ImmutableList.of(),
-            refs,
-            statisticsFiles,
-            partitionStatisticsFiles,
-            ImmutableList.of());
+        MetadataTestUtils.buildTestTableMetadataWithExampleValues(formatVersion)
+            .setCurrentSchemaId(7)
```

Comment:
we shouldn't really need any of these if they are in TestTableMetadataWithExampleValues since this is a ser/de test